README.md

```diff
@@ -26,17 +26,17 @@ This specification is currently being advanced through the [Meshery](https://lay
 
 ## Example
 
-See the [sample results](docs/assets/spec/readme/example/service%20mesh%20performance%20result.yaml) file.
+See the [sample results](docs/assets/perf/readme/example/service%20mesh%20performance%20result.yaml) file.
 
-<a href="docs/assets/spec/readme/service mesh performance example.gif"><img src="docs/assets/spec/readme/service mesh performance example.gif" /></a>
+<a href="docs/assets/perf/readme/service mesh performance example.gif"><img src="docs/perf/spec/readme/service mesh performance example.gif" /></a>
```

Comment:
Hi @JubayerJoy you made a mistake here, it should be 

src="docs/assets/perf/readme/service mesh performance example.gif" /></a> 👍🏽

Reply:
Wow! Your review skill is very impressive :clap:
I already have fixed in my latest [PR](https://github.com/layer5io/service-mesh-performance-specification/pull/84)

Thanks!

Reply:
Haha! 😆 Thanks a bunch. 🙌🏽